test/unit/bit.c

```diff
@@ -329,6 +329,68 @@ test_bit_copy_range(bool src_val)
 	footer();
 }
 
+static void
+random_bytes(int n, unsigned seed, unsigned char *out)
+{
+	srand(seed);
```

Comment:
Why do you call `srand(0)` here (`seed` is always 0 AFAICS). Let's drop it along with the `seed` argument. It might make sense to call `srand(time(NULL))` in the beginning of `main()` to randomize the input though.